pkg/utils/color.go

```diff
@@ -33,22 +32,23 @@ func getPadWidths(stringArrays [][]string) []int {
 	return padWidths
 }
 
-// GetHexColorValues returns the rgb values of a hex color
-func GetHexColorValues(v string) (r uint8, g uint8, b uint8, valid bool) {
-	if len(v) == 4 {
-		v = string([]byte{v[0], v[1], v[1], v[2], v[2], v[3], v[3]})
-	} else if len(v) != 7 {
-		return
+func IsValidHexValue(v string) bool {
```

Comment:
the `color` library has its own hex decoder so I've opted to use that, and this function is just for validating the HEX structure which I rolled myself because it's fairly simple